crates/polars-io/src/json/mod.rs

```diff
@@ -155,6 +161,101 @@ where
     }
 }
 
+pub trait BatchedWriter<W: Write> {
```

Comment:
can we reuse the `SinkWriter` trait here instead of adding a new trait?

Reply:
Since we changed the implementation to accept a JsonFormat as an input, BatchedWriter trait is not necessary anymore.

Reply:
Just as a comment, It would be great if we can expose SinkWriter to be used by other crates. This way we can move all SinkWriter implementations to the corresponding modules. 😄